backend/user-service/app/types/user.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+
+export const ROLE = {
```

Comment:
do you need this file?

Reply:
Resolved 437fbfc27a2f84c70a602983e4bd6c7a1753f4d6